TYPO3.Flow/Classes/TYPO3/Flow/Error/Debugger.php

```diff
@@ -175,14 +180,14 @@ protected static function renderObjectDump($object, $level, $renderProperties =
         $scope = '';
         $additionalAttributes = '';
 
-        if ($object instanceof \Doctrine\Common\Collections\Collection) {
-            return self::renderArrayDump(\Doctrine\Common\Util\Debug::export($object, 3), $level, $plaintext, $ansiColors);
+        if ($object instanceof Collection) {
+            return self::renderArrayDump(Debug::export($object, 3), $level, $plaintext, $ansiColors);
```

Comment:
Maybe since we also have `Debugger` from Flow, use something like `DoctrineDebug(ger)` or even keep the full name if it's used rarely - after all we're not gonna refactor Doctrine code ;)